projects/wave-core-new/assets/geoengine.svg

```diff
@@ -98,28 +98,44 @@
-       x="26.210703"
-       y="10.911593"
-       id="text27"><tspan
-         sodipodi:role="line"
-         id="tspan25"
-         x="26.210703"
-         y="10.911593"
-         style="fill:#6abf69;stroke:none;stroke-width:0.06614583;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:0.5;fill-opacity:1;">eng ne</tspan></text>
+    <g
+       aria-label="ge"
+       style="font-style:normal;font-weight:normal;font-size:10.58333302px;line-height:1.25;font-family:sans-serif;letter-spacing:0px;word-spacing:0px;fill:#388e3c;fill-opacity:1;stroke:none;stroke-width:0.06614583;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:0.5"
+       id="text23">
+      <path
+         d="m 8.5856698,7.9109631 q 0,-1.0335286 -0.4289143,-1.6019693 -0.4237468,-0.5684407 -1.1937256,-0.5684407 -0.7648112,0 -1.1937255,0.5684407 -0.4237468,0.5684407 -0.4237468,1.6019693 0,1.028361 0.4237468,1.5968018 0.4289143,0.5684411 1.1937255,0.5684411 0.7699788,0 1.1937256,-0.5684411 Q 8.5856698,8.9393241 8.5856698,7.9109631 Z M 9.5365162,10.15372 q 0,1.477946 -0.6562907,2.196249 -0.6562907,0.72347 -2.0102132,0.72347 -0.5012613,0 -0.9456786,-0.07752 -0.4444173,-0.07235 -0.8629964,-0.227376 V 11.84354 q 0.4185791,0.227376 0.8268229,0.335896 0.4082438,0.108521 0.8319905,0.108521 0.9353434,0 1.4004313,-0.490926 0.4650878,-0.485759 0.4650878,-1.472779 V 9.8539969 Q 8.2911142,10.365594 7.831194,10.618808 q -0.4599203,0.253215 -1.100708,0.253215 -1.0645345,0 -1.7156575,-0.81132 -0.651123,-0.8113203 -0.651123,-2.1497399 0,-1.3435872 0.651123,-2.1549071 0.651123,-0.81132 1.7156575,-0.81132 0.6407877,0 1.100708,0.2532145 0.4599202,0.2532145 0.7544758,0.7648112 V 5.0842624 h 0.9508464 z"
+         style="fill:#388e3c;fill-opacity:1;stroke:none;stroke-width:0.06614583;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:0.5"
+         id="path22" />
+      <path
+         d="m 16.445655,7.7404309 v 0.4650879 h -4.371826 q 0.06201,0.9818522 0.589111,1.4986165 0.532268,0.5115967 1.477946,0.5115967 0.54777,0 1.059367,-0.134359 0.516764,-0.1343585 1.023193,-0.4030759 v 0.8991699 q -0.511596,0.217041 -1.049031,0.330729 -0.537435,0.113688 -1.090373,0.113688 -1.384928,0 -2.196248,-0.806152 -0.806153,-0.8061524 -0.806153,-2.1807454 0,-1.4211019 0.764812,-2.2530924 0.769978,-0.8371582 2.072225,-0.8371582 1.167887,0 1.844848,0.7544759 0.682129,0.7493083 0.682129,2.041219 z M 15.494809,7.4613782 q -0.01034,-0.7803141 -0.43925,-1.245402 -0.423747,-0.4650879 -1.126546,-0.4650879 -0.795817,0 -1.276408,0.449585 -0.475423,0.4495849 -0.54777,1.2660725 z"
+         style="fill:#388e3c;fill-opacity:1;stroke:none;stroke-width:0.06614583;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:0.5"
+         id="path24" />
+    </g>
```

Comment:
Would it even be possible to override the fill values since we need lots of white in our app?

Reply:
I transformed the text to paths to make them work in the browser using Inkscape. I could change the colors too if that's what you mean.

Reply:
Maybe just add a second, white version into the repo and then it's fine.